src/components/ProductCard/index.jsx

```diff
@@ -2,13 +2,26 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 class ProductCard extends React.Component {
+  buttonAddProduct(productInfo, addProductToCart) {
+    return (
+      <button
+        type="button"
+        data-testid="product-add-to-cart"
+        onClick={ () => { addProductToCart(productInfo); } }
+      >
+        Adicionar Produto
+      </button>
+    );
+  }
+
   render() {
-    const { productInfo } = this.props;
+    const { productInfo, addProductToCart } = this.props;
     return (
       <div className="prodContainer" data-testid="product">
         <img src={ productInfo.thumbnail } alt="product img" />
-        <h3>{productInfo.title}</h3>
+        <h3 data-testid="shopping-cart-product-name">{productInfo.title}</h3>
```

Comment:
```suggestion
        <h3>{productInfo.title}</h3>
```
Será que o data-testid é realmente necessário aqui? Notei que ele já aparece no shopping cart.